deploy/helm/sumologic/values.yaml

```diff
@@ -4517,6 +4539,13 @@ opentelemetry-operator:
 
   manager:
     env:
+    resources:
+      limits:
+        cpu: 250m
+        memory: 512Mi
+      requests:
+        cpu: 150m
+        memory: 256Mi
```

Comment:
Are these based on anything specific? I'd always like to keep requests as low as possible for services that don't really need to scale.

Reply:
Yes, we observed on few deployments with opentelemetry-operator enabled that manager pod is crashing becuase of too low resources.